_pages/oidc.md

```diff
@@ -54,9 +54,10 @@ Consistent with the specification, login.gov provides a JSON endpoint for OpenID
 The authorization endpoint handles authentication and authorization of a user. To present the login.gov authorization page to a user, direct them to the `/openid_connect/authorize` endpoint with the following parameters:
 
 * **acr_values**
-  The Authentication Context Class Reference values used to specify the LOA (level of assurance) of an account, either LOA1 or LOA3. This and the `scope` determine which [user attributes]({{ site.baseurl }}/attributes/) will be available in the [user info response](#user-info-response). The possible parameter values are:
+  The Authentication Context Class Reference values used to specify the LOA (level of assurance) of an account, either LOA1, LOA3, or IAL0 (also known as IALMAX). If IALMAX is chosen the request will return IAL1 data if a user is not verified or IAL2 data if a user is verified. This and the `scope` determine which [user attributes]({{ site.baseurl }}/attributes/) will be available in the [user info response](#user-info-response). The possible parameter values are:
     - `http://idmanagement.gov/ns/assurance/loa/1`
     - `http://idmanagement.gov/ns/assurance/loa/3`
+    - `http://idmanagement.gov/ns/assurance/ial/0` (also known as IALMAX)
```

Comment:
+1 to what I think @porta-antiporta was asking, the code also allows `/ial/1` and `ial/2`, should we list those here?

Reply:
Old PR that had added it #89

Reply:
weird.  so it somehow got removed?

Reply:
We removed it when we had to roll back the change (the initial implementation to support IAL wasn't backwards compatible so it broke RRB). I suspect we may have omitted restoring it when we fixed the issue.

Would you be able to merge the changes in #89 to this branch? (Is that allowed in our SCM scheme?)

Reply:
sure